jetty-core/jetty-ee/src/test/java/org/eclipse/jetty/ee/TestableWebAppClassLoaderContext.java

```diff
@@ -25,19 +25,19 @@
 
 public class TestableWebAppClassLoaderContext implements WebAppClassLoader.Context
 {
-    private final ResourceFactory resourceFactory;
-    private boolean parentLoaderPriority = false;
-    private ClassMatcher protectedClassMatcher;
-    private ClassMatcher hiddenClassMatcher;
-    private List<Resource> extraClasspath;
-
-    public static void initEnvironment()
+    static
```

Comment:
I'm not as keen on this change. If this is now static then an individual test case can't use this class differently.

Reply:
So far, this class is only used by one test class which always calls `initEnvironment()` in a thread-unsafe way. IMHO we should postpone that static block refactoring to the day we start using `TestableWebAppClassLoaderContext` differently.